test/src/linux/snapTest.ts

```diff
@@ -294,3 +294,28 @@ test.ifDevOrLinuxCi(
     },
   })
 )
+
+test.ifDevOrLinuxCi(
```

Comment:
Perhaps this test should include the `compression` field, then we can check that it is still undefined but that it is also included in `args`?